docs/ecosystem/defi-liquidity/faq.md

```diff
@@ -49,7 +63,7 @@ Stablecoins can be traded on major Flow-based decentralized exchanges (DEXs) lik
 <Details summary="How can I earn yield on stablecoins on Flow?">
 You can earn yield through:
 
-- Lending Platforms - Supply stablecoins on [IncrementFi][3] & [MoreMarkets][4] to earn interest.
+- Lending Platforms - Supply stablecoins on [IncrementFi][3], [Sturdy Finance][19] & [MoreMarkets][4] to earn interest.
 - Liquidity Pools - Provide liquidity on [IncrementFi][5] or [KittyPunch][6] to earn trading fees and farm LP tokens.
 - Yield Aggregators (Coming soon) - Use [KittyPunch][7] to automate stablecoin yield strategies.
```

Comment:
Lets add a link to flowverse with a DeFi filter for them to explore more: https://www.flowverse.co/?categories=defi